compiler/src/utils/warnings.re

```diff
@@ -105,7 +105,7 @@ let message =
     )
   | NonClosedRecordPattern(s) =>
     "the following fields are missing from the record pattern: " ++ s
-  | EqualWasmUnsafe => "it looks like you are using Pervasives.(==) on two unsafe Wasm values here.\nThis is generally unsafe and will cause errors. Use `WasmI32.eq`, `WasmI64.eq`, `WasmF32.eq`, or `WasmF64.eq` instead.";
+  | FuncWasmUnsafe(func) => "it looks like you are using " ++ func ++ " on two unsafe Wasm values here.\nThis is generally unsafe and will cause errors. Use `WasmI32.eq`, `WasmI64.eq`, `WasmF32.eq`, or `WasmF64.eq` instead.";
```

Comment:
```suggestion
  | FuncWasmUnsafe(func) => "it looks like you are using " ++ func ++ " on two unsafe Wasm values here.\nThis is generally unsafe and will cause errors. Use the function from `WasmI32`, `WasmI64`, `WasmF32`, or `WasmF64` instead.";
```

Reply:
@phated I fixed this independently. Feel free to re-suggest another wording if you don't like what I changed it to.